worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/WorkflowManager.java

```diff
@@ -84,6 +93,10 @@ public Workflow get(final String workflowName){
                 } catch (final IOException e) {
                     throw new RuntimeException("Could not obtain workflow-control.js");
                 }
+                if (contextScriptFile != null && contextScriptFile.exists()) {
+                    final String contextScriptFileContent = FileUtils.readFileToString(contextScriptFile, StandardCharsets.UTF_8);
+                    stringBuilder.append(contextScriptFileContent);
+                }
```

Comment:
Is it correct to append context script to the end of the file rather than to prepend it to be beginning?

Reply:
It is by mistake. Updated to proper flow.